packages/init/test/SliceMachineInitProcess-createNewRepository.test.ts

```diff
@@ -61,19 +61,23 @@ const mockPrismicAPIs = async (
 	return { prismicAuthLoginResponse };
 };
 
-const loginWithStdin = async (
+const loginAndCreateRepositoryWithStdin = async (
 	prismicAuthLoginResponse: PrismicAuthLoginResponse,
 ) => {
 	const stdin = mockStdin();
 
+	// @ts-expect-error - Accessing protected method
+	const promise = initProcess.createNewRepository();
```

Comment:
Is it common practice?

Reply:
I copy this from another place in tests:

<img width="893" alt="Screenshot 2024-02-08 at 15 50 49" src="https://github.com/prismicio/slice-machine/assets/19946868/4040f20e-42ca-43ae-b84d-f4b1603edc4a">

I think it's fine to fix our problem, but let me know if you think of something better

Reply:
I thought there was a way to specify the ts error like in eslint, but checked and it's not. Fine then!